twilio.go

```diff
@@ -125,18 +127,25 @@ type Meta struct {
 	URL             *string `json:"url"`
 }
 
+type ClientCredentialProvider struct {
```

Comment:
Need documentation here and for ClientParams below. When do I want to use each one? How can I figure out where to go in the dashboard to retrieve the appropriate values?